readme.md

```diff
@@ -75,6 +75,59 @@ Or to be more specific:
 localStorage['debug'] = 'browser-monkey';
 ```
 
+# mount
+
+Typically you will need to mount your application into the DOM before running your tests.
+
+Browser monkey comes with a handy way of doing this for popular web frameworks
+
+**hyperdom**
+where YourHyperdomApp is a class that has a render method. [see here](test/app/hyperdom.jsx) for an example
+
+```js
+var monkey = require('browser-monkey/hyperdom')
+  .withApp(() => new YourHyperdomApp())
+  .start()
+```
+
+**angular**
+where YourAngularApp is a class with fields 'directiveName' and 'moduleName' [see here](test/app/angular.js) for an example
+
+```js
+var monkey = require('browser-monkey/angular')
+  .withApp(() => new YourAngularApp())
```

Comment:
yeah true, honestly I am not even sure that is the best way of loading in the angular app, it is just how we have done it before, it is a bit rubbish because you need to make a special wrapper directive which I don't think people usually do